bindings/python/pymongoarrow/__init__.py

```diff
@@ -29,8 +29,10 @@ def _parse_version(version):
 
 try:
     from pymongoarrow.lib import libbson_version
-except ImportError:
-    warnings.warn("This library has not been compiled")
+except ImportError as e:
+    warnings.warn(
+        f"Could not find compiled pymongoarrow.lib extension, please install from source or report the following traceback on the issue tracker:\n{e}"
+    )
```

Comment:
line length

Reply:
Should we include the full traceback in the warning?

Reply:
Sounds good